apps/condo/jest.setupSpec.js

```diff
@@ -2,6 +2,8 @@ const falsey = require('falsey')
 
 const EXTRA_LOGGING = falsey(process.env.DISABLE_LOGGING)
 
+jest.setTimeout(15000)
```

Comment:
Should it be modified right here? May be it's possible to specify it only for a specific test case?

Reply:
This value is set to run tests in ci. I don't think it will be redundant

Reply:
This is also specified in `jest.setupTest.js`, but I think 60 seconds will be a lot

Reply:
It's better don't change it. We use jest options for that